src/AntDesign.Pro/Pages/Account/Center/Components/Applications/Applications.razor

```diff
@@ -35,16 +33,25 @@
 
 @code
 {
-    private static readonly RenderFragment Download = @<Tooltip Title="@("下载")">
-                                                           <Icon Type="download" Theme="outline"/>
-                                                       </Tooltip>;
+
+    private static RenderFragment<(ForwardRef refBack, string type)> _icon => (data) =>@<Icon RefBack="@data.refBack" Type="@data.type" Theme="outline" />;
+
+    private static RenderFragment<(string tooltipTitle, RenderFragment<(ForwardRef, string)> icon, string iconType)> _tooltipTemplate => (data) => __builder =>
+    {
+        __builder.OpenComponent<AntDesign.Tooltip>(0);
+        __builder.AddAttribute(1, "Title",
+            Microsoft.AspNetCore.Components.CompilerServices.RuntimeHelpers.TypeCheck<OneOf.OneOf<System.String, RenderFragment, MarkupString>>(data.tooltipTitle));
+        __builder.AddAttribute(2, "Unbound", (RenderFragment<ForwardRef>)((context) =>
+            data.icon((context, data.iconType)) ));
+            __builder.CloseComponent();
+    };
```

Comment:
I think this is too complicated for the average user...

Reply:
Heh, that is actually a compliment 😉 . But this is why I asked [this](https://github.com/dotnet/aspnetcore/issues/29671). If there is no other (cleaner) way , then I will do another commit with the second option.

Reply:
So, event though the blazor guys replied to my question and actually solved my problem, this does not solve this situation. The moment I tried to pass a bit more info than just the context, the whole thing broke down. So I cannot do something like this:
```razor
@_tooltipTemplate1((new ForwardRef(), "下载", "download")),

@code{
    private static RenderFragment<(ForwardRef context, string tooltipTitle, string iconType)> _tooltipTemplate => data =>
    @<Tooltip Title="@data.tooltipTitle" RefBack="@data.context">
        <Unbound>
            <Icon RefBack="@data.context" Type="@data.iconType" Theme="outline" />
        </Unbound>
     </Tooltip>;
}
```
because it just does not work (tooltip does not show). It seems to only work when generic type is just `ForwardRef` not a tuple with whatever other types I'd like to pass. I even tried a custom object instead of a tuple. Same result (no tooltip). Therefore only this seems to work:
```razor
@code {
    private static RenderFragment<ForwardRef> _tooltipDownload => context =>
    @<Tooltip Title="@("下载")">
        <Unbound>
            <Icon RefBack="@context" Type="download" Theme="outline" />
        </Unbound>
    </Tooltip>;

    private static RenderFragment<ForwardRef> _tooltipEdit => context =>
    @<Tooltip Title="@("编辑")">
        <Unbound>
            <Icon RefBack="@context" Type="edit" Theme="outline" />
        </Unbound>
    </Tooltip>;

    private static RenderFragment<ForwardRef> _tooltipShareAlt => context =>
    @<Tooltip Title="@("分享")">
        <Unbound>
            <Icon RefBack="@context" Type="share-alt" Theme="outline" />
        </Unbound>
    </Tooltip>;

    private static readonly IList<RenderFragment> Actions = new List<RenderFragment>
    {
        _tooltipDownload(new ForwardRef()),
        _tooltipEdit(new ForwardRef()),
        _tooltipShareAlt(new ForwardRef()),
        @<Dropdown><Icon Type="ellipsis" Theme="outline" /></Dropdown>        
    };
}
```
This is obviously much more verbose. But it should be simpler to understand. Do you want me to go with this solution? Or shall I revert to bound to div `ChildContent`?

Reply:
I think it would be easier for users to use div `ChildContent` if it didn't affect the effect 😂.
Anyway, thanks for your work.